backbone.js

```diff
@@ -112,42 +101,46 @@
 
       // Remove all callbacks for all events.
       if (!name && !callback && !context) {
-        this._events = void 0;
+        var listenerIds = _.keys(this._listeners);
+        for (var i = 0, l = listenerIds.length; i < l; i++) {
```

Comment:
The advantage to using the util libs methods is it includes iteration fixes that you wouldn't get otherwise.